docs/docs/guides/managing-appeals.md

```diff
@@ -2,18 +2,23 @@
 
```

Comment:
@rahmatrhd Let's add the context of what this page talks about. This will also help to create a visual gap between two continuous headings. We can follow the same principle across.